### PR TITLE
update node-labeller documentation

### PR DESCRIPTION
**Pull request by ksimon1**
update node-labeller documentation
/hold 
Merge only after kubevirt PR https://github.com/kubevirt/kubevirt/pull/4916 is merged
Signed-off-by: Karel Simon <ksimon@redhat.com>

### CODE REVIEW

**Comment by mazzystr** [comment]
@ksimon1  We have bad timing on your contribution.  The user-guide migration to mkdocs markdown engine and restructuring the page hierarchy completed today.

Your pull req now needs a rebase.  The rebase will be difficult. `creation/devices.md` has been renamed to `docs/virtual_machines/virtual_hardware.md`.  You will need to integrate this diff into the new file.  Another option is pull master, recreate your branch, recreate the content in the new file, create new pull req.  Yet another option is for me to do it.   The con to that is you don't get the GitHub credit.

I'm happy to discuss further in [slack](https://kubernetes.slack.com/archives/C8ED7RKFE)

**Comment by mazzystr** [comment]
@@ksimon1 Thanks for going through the rebase journey.  I will remove the `do-not-merge/hold` label and proceed with normal peer review.

File: docs/virtual_machines/virtual_hardware.md [hunk 1]
```diff
@@ -149,47 +149,48 @@ be scheduled on the node which can support VM cpu model and features.
...
-([kubevirt-ssp-operator](https://github.com/MarSik/kubevirt-ssp-operator)),
-which will install node-labeller + all available plugins.
-
-Cpu-nfd-plugin uses libvirt to get all supported cpu models and cpu
-features on host and Node-labeller create labels from cpu models. Then
-Kubevirt can schedule VM on node which has support for VM cpu model and
+To properly label the node, user can use Kubevirt Node-labeller, which creates all 
+neccessary labels or create node labels by himself.
+
+Kubevirt node-labeller creates 3 types of labels: cpu models, cpu features, kvm info.
```
Comment (mazzystr) [inline docs/virtual_machines/virtual_hardware.md:155]: The last item in a comma separated list get a `, and item`

File: docs/virtual_machines/virtual_hardware.md [hunk 2]
```diff
@@ -149,47 +149,48 @@ be scheduled on the node which can support VM cpu model and features.
...
+      name: kubevirt
+      namespace: kubevirt
+    spec:
+      ...
+      configuration:
+        minCPUModel: "Penryn"
+        obsoleteCPUModels:
           - "486"
           - "pentium"
-        minCPU: "Penryn"
```
Comment (mazzystr) [inline docs/virtual_machines/virtual_hardware.md:179]: Please encapsulate this code block with ` ```console`.  All these bare code blocks have been showing up in lintting.  We need to get that fixed.

**Comment by mazzystr** [comment]
@ksimon1 just a couple nits from me then +1

**Review by mazzystr — CHANGES_REQUESTED** [review]

**Review by mazzystr — CHANGES_REQUESTED** [review]
Fix a couple nits, rebase then you've got your lgtm/approve tag :)

**Review by mazzystr — CHANGES_REQUESTED** [review]
@ksimon1 Fix a couple of nits and rebase then this will be ready for merge.  Thanks

**Comment by ksimon1** [comment]
This can't be merged yet. This PR has to be merged before https://github.com/kubevirt/kubevirt/pull/4916

**Comment by mazzystr** [comment]
@ksimon1 ping.  We're getting close to 30 days in-progress on this pull req.  Are you still wanting to merge this?

**Comment by ksimon1** [comment]
@mazzystr definitely. Still waiting for this PR https://github.com/kubevirt/kubevirt/pull/4916 to be merged and then this PR can be merged.

**Comment by mazzystr** [comment]
@ksimon1  I applied the do-not-merge/hold label.  Can you rebase and fix small items so that this pull req will be ready to go?

**Comment by ksimon1** [comment]
/hold cancel
@mazzystr this PR is ready to be merged

**Pull request merged**